src/dragonball/src/dbs_virtio_devices/Cargo.toml

```diff
@@ -53,4 +53,4 @@ virtio-mem = ["virtio-mmio"]
 virtio-balloon = ["virtio-mmio"]
 vhost = ["virtio-mmio", "vhost-rs/vhost-user-master", "vhost-rs/vhost-kern"]
 vhost-net = ["vhost", "vhost-rs/vhost-net"]
-vhost-user = ["vhost"]
\ No newline at end of file
+vhost-user-fs = ["vhost"]
```

Comment:
It would be better to define:
```
vhost-user = ["vhost"]
vhost-user-fs = ["vhost-user"]
```
Then it's flexible to support `vhost-user-blk` etc in future.